card/mod/05_standard/spec/set/all/account_spec.rb

```diff
@@ -124,7 +124,7 @@
       expect(@read_rules.member?(Card.fetch('*all+*read').id)).to be_truthy
     end
 
-    it "9 more should apply to Joe Admin" do
+    it "14 more should apply to Joe Admin" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.